### PR TITLE
FEAT: add additional metadata to pyproject, description, version, authors

### PR DESCRIPTION
**Pull request by dni**
add additional metadata to pyproject, description, version, authors

### CODE REVIEW

**Comment by callebtc** [comment]
Do we need to update this manually with every new release? :(

**Comment by dni** [comment]
there is a way around this, but at the moment, yes.
i can look into creating a pypi package, when i do this, i am going to add version automatically.

but for now this is better than nothing.

File: pyproject.toml [hunk 1]
```diff
@@ -1,8 +1,12 @@
 [tool.poetry]
 name = "lnbits"
-version = "0.1.0"
-description = ""
-authors = ["matthewcroughan <matt@croughan.sh>"]
+version = "0.9.5.3"
+description = "LNbits, free and open-source lightning-network wallet/accounts system"
```
Comment (callebtc) [inline pyproject.toml:4]:
```suggestion
description = "LNbits, free and open-source Lightning wallet and accounts system."
```

**Comment by arcbtc** [comment]
We should add everyone or no one here I thnk
image

**Comment by callebtc** [comment]
Agree that we should just have lnbits as an author, otherwise we will be busy keeping this list up to date. Whoever wants to see the actual authors, will look on github anyway, I think.

File: pyproject.toml [hunk 2]
```diff
@@ -1,8 +1,12 @@
...
-version = "0.1.0"
-description = ""
-authors = ["matthewcroughan <matt@croughan.sh>"]
+version = "0.9.5.3"
+description = "LNbits, free and open-source Lightning wallet and accounts system."
+authors = [
+    "benarc <ben@lnbits.com>",
+    "dni <dni@lnbits.com>",
+    "matthewcroughan <matt@croughan.sh>"
+]
```
Comment (callebtc) [inline pyproject.toml:9]:
```suggestion
authors = [
    "lnbits <lnbits@lnbits.com>",
]
```

Does this address exist @arcbtc?

**Comment by dni** [comment]
i also agree, lets check if email is valid and then merge ;)

File: pyproject.toml (hunk 2 above)
Comment (dni) [reply pyproject.toml]: 08c485e

File: pyproject.toml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "lnbits"
-version = "0.1.0"
-description = ""
-authors = ["matthewcroughan <matt@croughan.sh>"]
+version = "0.9.5.3"
+description = "LNbits, free and open-source Lightning wallet and accounts system."
+authors = ["lnbits <lnbits@pm.me>"]
```
Comment (callebtc) [inline pyproject.toml:5]:
```suggestion
authors = ["Alan Bits <alan@lnbits.com>"]
```

**Pull request merged**